pytest_splunk_addon_ui_smartx/components/message_tray.py

```diff
@@ -0,0 +1,101 @@
+# SPDX-FileCopyrightText: 2020 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from .base_component import BaseComponent, Selector
+from selenium.webdriver.common.by import By
+
+class MessageTray(BaseComponent):
+    """
+    Component: MessageTray
+    Base class of Input & Configuration table
+    """
+    def __init__(self, browser, container, mapping=dict()):
```

Comment:
Please remove the container from the argument.  I don't think we should expect a user to add `container` as there always be only 1 MessageTray.

Reply:
Resolved in this commit: 43bff98